contracts/src/starknet/rewaave/token_bridge.cairo

```diff
@@ -179,3 +184,19 @@ func handle_deposit{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_chec
 
     return ()
 end
+
+@external
+func mint_rewards{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+        recipient : felt, amount : Uint256):
+    # get the address of the ETHStaticAToken
+    let (l2_token) = get_caller_address()
+    # Verify that it's a valid token by checking for its counterpart on l1
+    let (l1_token) = l2_token_to_l1_token.read(l2_token)
+    with_attr error_message("L1 token {l1_token} not found"):
```

Comment:
This should report that the L1 token for the l2 token isn't found. Right now it's just say "L1 token 0x0 not found"